norse/torch/functional/lsnn.py

```diff
@@ -51,7 +50,7 @@ class LSNNParameters(NamedTuple):
 
     tau_syn_inv: torch.Tensor = torch.as_tensor(1.0 / 5e-3)
     tau_mem_inv: torch.Tensor = torch.as_tensor(1.0 / 1e-2)
-    tau_adapt_inv: torch.Tensor = torch.as_tensor(1.0 / 700)
+    tau_adapt_inv: torch.Tensor = torch.exp(torch.as_tensor(-1.0 / 800))
```

Comment:
where does this come from?

Reply:
https://arxiv.org/pdf/1901.09049.pdf (26)
I liked the way they define the decay as proportional to the number of timesteps one would like to retain information. Meaning, that `exp^{-1 \over 800}` means that the decay roughly happens over 800 steps. This argument is taken from https://arxiv.org/pdf/1803.09574.pdf (2)

Reply:
Ok sure, but then you need to use ```exp(-dt/800)``` otherwise this doesn't work I believe.